install/tests/Makefile

```diff
@@ -398,7 +402,12 @@ wait_time_azure := 300
 delete-cm-setup: sleeptime=$(if $(time_to_sleep_$(cloud)),$(time_to_sleep_$(cloud)),${time_to_sleep})
 delete-cm-setup: waittime=$(if $(wait_time_$(cloud)),$(wait_time_$(cloud)),${wait_time})
 delete-cm-setup:
-	sleep ${waittime} && kubectl --kubeconfig=${KUBECONFIG} delete pods --all -n cert-manager && sleep ${sleeptime};
+	@echo "Waiting ${waittime} for cert-manager to become ready"
+	kubectl --kubeconfig=${KUBECONFIG} --namespace cert-manager wait --for=condition=Available deployment/cert-manager --timeout=${waittime}s
+	kubectl --kubeconfig=${KUBECONFIG} delete pods --all -n cert-manager
+	@echo Waiting for up to ${sleeptime} seconds for cert-manager to become ready
+	kubectl --kubeconfig=${KUBECONFIG} --namespace gitpod wait --for=condition=Ready certificate/https-certificates --timeout ${sleeptime}s || true
```

Comment:
Once the certificate becomes ready it has to wait a bit more for Gitpod to become ready. Looks like the job you triggered(https://werft.gitpod-dev.com/job/gitpod-custom-alt-installer-tests-certificate-backup-resto.0) failed because it didn't wait for services to come up. I think there should still be a wait time, definitely reduced. Maybe like 2-3 minutes.